bondjimbond/islandora_downloadable_image
# Moremodels

Addresses #2 - Config page lists all Islandora CModels, allows user to select which ones will generate a download block.

**To test**
- Disable and uninstall Islandora Downloadable Image
- Check out the moremodels branch
- Enable Islandora Downloadable Datastreams, then disable and uninstall it (to clear out some leftover variables from Islandora Downloadable Image)
- Re-enable Islandora Downloadable Datastreams, and enable the Dastream Download Block

Only currently configured to work with the following CModels:
- Large Image
- Video
- Audio
- Newspaper Issue
- Book

Future development will make the DSIDs for these CModels configurable. Will also determine whether any other CModels need to be included -- if not, re-do the form to limit selectable CModels to the ones that are actually usable.
